src/components/ResetButton/ResetButton.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+function ResetButton() {
+  return (
+    <button
+      className="filter__button-reset"
+      type="button"
+      onClick="onChange"
```

Comment:
Passing a string as the callback smells fishy. Just don't set the `onClick` here if it doesn't do anything yet.